OJP_Locations.xsd

```diff
@@ -343,6 +341,11 @@
 					<xs:documentation>Rough estimate of the travel duration from the specified location to this exchange point.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="WaitDuration" type="xs:duration" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Duration needed at this exchange point to change from one service to another. If a journey planning orchestrator puts together a trip at this exchange point, it hat to take care, that feeder and fetcher are at leat this duration apart. This is not true, if either feeding or fetching </xs:documentation>
```

Comment:
"it ha_**s**_ to take care that feeder and fetcher are at lea_**s**_t this duration apart" instead of "it hat to take care, that feeder and fetcher are at leat this duration apart"
Is there text missing to complete the sentence after "This is not true, if either feeding or fetching "?

Reply:
Sorry, yesterday I was so annoyed, that nothing worked, that I didn't check the annotations anymore.

Reply:
Solved in new commit.